source/MaterialXRuntime/Private/PrvNode.h

```diff
@@ -84,6 +76,14 @@ class PrvNode : public PrvAllocatingElement
     static void disconnect(const RtPort& source, const RtPort& dest);
 
 protected:
+    // Constructor creating a node with a fixed interface
+    // This is the constructor to use for ordinary nodes.
+    PrvNode(const RtToken& name, const PrvObjectHandle& nodedef, RtObjType objType = RtObjType::NODE);
```

Comment:
If one of these if for nodes and the other nodegraphs, why even allow other objTypes?

Reply:
It's a good point. This was an attempt to try and clarify in the code what the intent with the two constructs are, but it's probably just more confusing. So I will remove it. The constructors are protected now anyway and it's the private API so it's only us that will use these constructors.